tests/Jenkinsfile

```diff
@@ -8,7 +8,11 @@ pipeline {
         booleanParam(name: 'FETCH_IMAGE', defaultValue: false)
         text(name: 'PYTEST_ARGS', defaultValue: """-sv \\
 tests/integration_tests/tests/agentless_tests/test_workflow.py \\
-tests/integration_tests/tests/agentless_tests/test_resume.py""")
+tests/integration_tests/tests/agentless_tests/test_resume.py \\
+tests/integration_tests/tests/agentless_tests/test_rest_api.py \\
+tests/integration_tests/tests/agentless_tests/test_rest_service_list_filters.py \\
+tests/integration_tests/tests/agentless_tests/test_rest_service_pagination.py \\
+tests/integration_tests/tests/agentless_tests/test_rest_service_sort.py""")
```

Comment:
we need a better way than this

Reply:
I think once we fix all tests we can update that or what define all these tests in a separate  file for int-tests